clients/webassembly/js-example/index.html

```diff
@@ -15,8 +15,17 @@
   <p>
     <label>Recipient: </label><input size="85" type="text" id="recipient" value="">
   </p>
-  <p>
+  <!-- <p>
     <label>Message: </label><input type="text" id="message" value="Hello mixnet!">
+  </p> -->
+  <p>
+    <label>Telegram @: </label><input type="text" id="telegram" value="">
+  </p>
+  <p>
+    <label>HAL address: </label><input type="text" id="hal_address" value="">
```

Comment:
This is one line I wonder about - if this example is for milhon, is it not the case that the currency would be more punk? 

There are a couple HAL-related things in the code that I expect would need to change.